### PR TITLE
Remove es6 worker and update dependencies

### PR DESCRIPTION
**Pull request by Swaagie**
## Summary

Remove the ES6 worker as an option for the factory. Publishing modules that only contain untranspiled code is considered a bad practice. Remove the builder to not promote this type of workflow.

**nb: should be released as new major, external API changes**

## Changelog

Updated

## Test Plan

Tests were adjusted and `npm test` works

### CODE REVIEW

**Comment by 3rd-Eden** [comment]
> Publishing modules that only contain untranspiled code is considered a bad practice. 

ACKTUALLY. As it's using babel, it's not untranspiled code.

File: package.json [hunk 1]
```diff
@@ -29,38 +29,35 @@
...
   "dependencies": {
     "@wrhs/extract-config": "^1.0.0",
-    "async": "~2.1.4",
-    "babel-core": "~6.21.0",
-    "babel-preset-es2015": "~6.18.0",
-    "babel-preset-react": "~6.16.0",
-    "bl": "~1.2.0",
-    "browserify": "~13.3.0",
-    "clean-css": "~3.4.23",
+    "async": "~3.1.0",
```
Comment (msluther) [inline package.json]: Since you're touching them... Do we want to start using `^` here or stick with `~`?

File: test/factory.test.js [hunk 2]
```diff
@@ -439,9 +400,9 @@ describe('Factory', function () {
 
         assume(factory.output).to.be.an('object');
         assume(factory.output['base.min.css'].content).to.be.instanceof(Buffer);
-        assume(factory.output['base.min.css'].content.toString()).to.include('span{margin:0;font-size:12px;color:#FFF}');
+        assume(factory.output['base.min.css'].content.toString()).to.include('span{margin:0;font-size:12px;color:#fff}');
```
Comment (msluther) [inline test/factory.test.js]: lol. Are lowercase `f`s smaller? 😂

**Comment by msluther** [comment]
Does [`extract-config`](https://github.com/warehouseai/extract-config) need touched for this change?

File: test/factory.test.js (hunk 2 above)
Comment (Swaagie) [reply test/factory.test.js]: /shrug :) change brought in by clean-css@4, they might have reasons or just OCD

**Comment by Swaagie** [comment]
I don't think extract-config needs to be touched. It seems all values are just read and not filtered against predefined values. E.g. filter `build` against worker types etc.

**Pull request merged**